bin/_config.sh

```diff
@@ -60,25 +58,28 @@ function _docker_run() {
         i=$(( i+1 ))
     done
 
-    # Check that minishift is running
-    if ${OC_LOGIN} && ! minishift status | grep Running > /dev/null ; then
-        echo "Error: minishift is not running. Please start it first."
+    # Check that the current user is already logged in OpenShift
+    if ! oc whoami &> /dev/null; then
+        echo "Error: you need to login to an OpenShift server first."
         exit 1
     fi
 
-    # Check that the current user is already logged in minishift
-    if ${OC_LOGIN} && ! oc whoami &> /dev/null; then
-        echo "Error: you need to login to minishift first."
-        exit 1
-    fi
+    [[ "$USE_TTY" == "false" ]] && TTY_OPTION="" || TTY_OPTION="t"
```

Comment:
I would prefer: `TTY_OPTION="-t"` and adding a space between `-i` and the `${TTY_OPTION}` in the `docker run` invocation.

Reply:
ok